src/display/node_stream.js

```diff
@@ -362,8 +362,13 @@ class PDFNodeStreamRangeReader extends BaseRangeReader {
 class PDFNodeStreamFsFullReader extends BaseFullReader {
   constructor(stream) {
     super(stream);
+
     let path = decodeURI(this._url.path);
 
+    if (/^file:\/\/\/[a-zA-Z]:\//.test(this._url.href)) {
+      path = path.replace(/^\//, '');
```

Comment:
Can you add a comment above the `if` statement for why this is necessary, i.e., what the exact problem on Windows is that this solves?